modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/js/components/RatingsSelectStars.js

```diff
@@ -86,10 +100,15 @@ export default function RatingsSelectStars({
 							return (
 								<ClayDropDown.Item
 									active={label === score}
+									id={'rsd_' + index}
```

Comment:
I think is better to extract the `'rsd_'` to a variable/constant something like OPTION_ID_PREFIX outside of the component must work.

Reply:
Sounds good to me. 

@ambrinchaudhary Let me know if you want me to change it and send a new PR to liferay-lima or you are going to make changes directly on your branch and push them so we can keep this PR.